draw/line_style.go

```diff
@@ -39,6 +39,19 @@ var lineStyleChars = map[LineStyle]map[linePart]rune{
 		vAndRight:         '├',
 		vAndH:             '┼',
 	},
+	LineStyleDouble: {
```

Comment:
This looks great, thanks a lot!

Would you mind sending this in a separate pull request? A couple of reasons really:
1) this being a new feature - I would prefer to merge it into the devel branch instead of the master and follow a release cycle.
2) for the posterity - the pull request history or rollbacks become much cleaner when each pull request is focused on only one bug / feature / etc.
3) we could iterate on this change a bit more without blocking your original PR regarding the lint issues. This new feature (new line style) could benefit from some test coverage. An advice I got a long time ago was to accompany each code change with at least minimal corresponding change in tests to "capture" the new feature in a working state.

I have no more comments on this PR without the new line style, so once we remove the line style from this scope - this is ready to go.

For (3) we could expand the test cases inside the border_test:

https://github.com/mum4k/termdash/blob/master/draw/border_test.go

Reply:
I've reverted the merge with the line-style. 

I'll check out that test file 👍

Reply:
Thank you.